Dockerfile

```diff
@@ -13,39 +13,16 @@ WORKDIR /app
 
 # Update and install necessary packages
 RUN apt-get update && apt-get install -y \
-	wget \
-	bzip2 \
-	ca-certificates \
-	&& rm -rf /var/lib/apt/lists/*
-
-# Install Mambaforge for the appropriate architecture
-RUN arch=$(uname -m) && \
-	if [ "${arch}" = "x86_64" ]; then \
-		wget -q "https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Linux-x86_64.sh" -O mambaforge.sh; \
-	elif [ "${arch}" = "aarch64" ]; then \
-		wget -q "https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Linux-aarch64.sh" -O mambaforge.sh; \
-	else \
-		echo "Unsupported architecture: ${arch}"; \
-		exit 1; \
-	fi && \
-	bash mambaforge.sh -b -p /opt/mambaforge && \
-	rm mambaforge.sh
-
-# Add Mambaforge to PATH
-ENV PATH=/opt/mambaforge/bin:$PATH
-
-# Create a new environment with Python 3.11
-RUN mamba create -n team3_env python=3.11 -y
-
-# Activate the new environment
-SHELL ["mamba", "run", "-n", "team3_env", "/bin/bash", "-c"]
+    wget \
+    bzip2 \
+    ca-certificates \
+    && rm -rf /var/lib/apt/lists/*
 
 # Copy requirements.txt into the container
 COPY requirements.txt /app/requirements.txt
 
 # Install Python packages from requirements.txt
-RUN mamba install --yes --file requirements.txt && \
-	mamba clean --all -f -y
+RUN pip install --no-cache-dir -r requirements.txt
```

Comment:
Should we use mamba or pip to install the requirements?

Reply:
I am not sure

Reply:
Oh yeah we have to use mamba right I sorry I forgot that